pkg/littlefs2/Makefile

```diff
@@ -8,10 +8,8 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 CFLAGS += -Wno-format
 
-# Disable debug printing
-ifneq ($(DEVELHELP),1)
-  CFLAGS += -DLFS_NO_DEBUG -DLFS_NO_WARN -DLFS_NO_ERROR
-endif
+# replace pkg supplied logging macros with RIOT's
+CFLAGS += -include $(PKG_DIR)/lfs_log.h
```

Comment:
```suggestion
CFLAGS += -I$(PKG_DIR)/lfs_log.h
```

Reply:
Those options do different things:
- `-I` adds a folder to the include search patch
- `-include` adds a file to be included as if by `#include "..."` to a compilation unit.

Reply:
Oh, my bad. Disregard my comment then.